posthog/queries/funnels/base.py

```diff
@@ -737,6 +751,16 @@ def _get_breakdown_select_prop(self) -> str:
                     table="groups", property_name=self._filter.breakdown, var="%(breakdown)s", column=properties_field
                 )
             basic_prop_selector = f"{expression} AS prop_basic"
+        elif self._filter.breakdown_type == "hogql":
```

Comment:
Should this function now be returning params as well?

Reply:
Not "now", as it gathers the params into `hogql_context.values`, which is shared/controlled outside the queries. At least that's how it is "now".